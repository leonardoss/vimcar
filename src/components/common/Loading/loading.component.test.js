import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure, render } from "enzyme";
import Loading from "./";

configure({ adapter: new Adapter() });

describe("Testing <Loading", () => {
  it("should render", () => {
    expect(tree()).toHaveLength(1);
  });
  it("should match the snapshot", () => {
    expect(tree()).toMatchSnapshot();
  });
});

const tree = () => render(<Loading />);
