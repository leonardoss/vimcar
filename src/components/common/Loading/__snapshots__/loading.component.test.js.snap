// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Loading should match the snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "loading_spinner-wrap",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "loading_spinner",
          "height": "20",
          "viewBox": "0 0 60 20",
          "width": "60",
          "xmlns": "http://www.w3.org/2000/svg",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "cx": "7",
              "cy": "15",
              "r": "4",
            },
            "children": Array [],
            "name": "circle",
            "namespace": "http://www.w3.org/2000/svg",
            "next": Object {
              "attribs": Object {
                "cx": "30",
                "cy": "15",
                "r": "4",
              },
              "children": Array [],
              "name": "circle",
              "namespace": "http://www.w3.org/2000/svg",
              "next": Object {
                "attribs": Object {
                  "cx": "53",
                  "cy": "15",
                  "r": "4",
                },
                "children": Array [],
                "name": "circle",
                "namespace": "http://www.w3.org/2000/svg",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "cx": undefined,
                  "cy": undefined,
                  "r": undefined,
                },
                "x-attribsPrefix": Object {
                  "cx": undefined,
                  "cy": undefined,
                  "r": undefined,
                },
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
              "x-attribsPrefix": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "cx": undefined,
              "cy": undefined,
              "r": undefined,
            },
            "x-attribsPrefix": Object {
              "cx": undefined,
              "cy": undefined,
              "r": undefined,
            },
          },
          Object {
            "attribs": Object {
              "cx": "30",
              "cy": "15",
              "r": "4",
            },
            "children": Array [],
            "name": "circle",
            "namespace": "http://www.w3.org/2000/svg",
            "next": Object {
              "attribs": Object {
                "cx": "53",
                "cy": "15",
                "r": "4",
              },
              "children": Array [],
              "name": "circle",
              "namespace": "http://www.w3.org/2000/svg",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
              "x-attribsPrefix": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "cx": "7",
                "cy": "15",
                "r": "4",
              },
              "children": Array [],
              "name": "circle",
              "namespace": "http://www.w3.org/2000/svg",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
              "x-attribsPrefix": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "cx": undefined,
              "cy": undefined,
              "r": undefined,
            },
            "x-attribsPrefix": Object {
              "cx": undefined,
              "cy": undefined,
              "r": undefined,
            },
          },
          Object {
            "attribs": Object {
              "cx": "53",
              "cy": "15",
              "r": "4",
            },
            "children": Array [],
            "name": "circle",
            "namespace": "http://www.w3.org/2000/svg",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "cx": "30",
                "cy": "15",
                "r": "4",
              },
              "children": Array [],
              "name": "circle",
              "namespace": "http://www.w3.org/2000/svg",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "cx": "7",
                  "cy": "15",
                  "r": "4",
                },
                "children": Array [],
                "name": "circle",
                "namespace": "http://www.w3.org/2000/svg",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "cx": undefined,
                  "cy": undefined,
                  "r": undefined,
                },
                "x-attribsPrefix": Object {
                  "cx": undefined,
                  "cy": undefined,
                  "r": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
              "x-attribsPrefix": Object {
                "cx": undefined,
                "cy": undefined,
                "r": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "cx": undefined,
              "cy": undefined,
              "r": undefined,
            },
            "x-attribsPrefix": Object {
              "cx": undefined,
              "cy": undefined,
              "r": undefined,
            },
          },
        ],
        "name": "svg",
        "namespace": "http://www.w3.org/2000/svg",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
          "height": undefined,
          "viewBox": undefined,
          "width": undefined,
          "xmlns": "http://www.w3.org/2000/xmlns/",
        },
        "x-attribsPrefix": Object {
          "class": undefined,
          "height": undefined,
          "viewBox": undefined,
          "width": undefined,
          "xmlns": "",
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
